web/src/components/NewsListItem.tsx

```diff
@@ -16,7 +16,15 @@ const StyledLink = styled(Link)`
 `
 const ReadMore = styled.div<{ $type: NewsType }>`
   align-self: flex-end;
-  color: ${({ theme, $type }) => ($type === LOCAL_NEWS_TYPE ? theme.colors.themeColor : theme.colors.tunewsThemeColor)};
+  color: ${({ theme, $type }) => {
+    if (theme.isContrastTheme) {
+      return theme.colors.themeColor
+    }
+    if ($type === LOCAL_NEWS_TYPE) {
+      return theme.colors.themeColor
+    }
```

Comment:
```suggestion
    if (theme.isContrastTheme || $type === LOCAL_NEWS_TYPE) {
      return theme.colors.themeColor
    }
```

Reply:
:monkey: I missed it, they have the same color